CODEOWNERS

```diff
@@ -179,7 +179,6 @@
 /dts/bindings/*/sifive*                   @mateusz-holenko @kgugala @pgielda @nategraff-sifive
 /dts/bindings/*/litex*                    @mateusz-holenko @kgugala @pgielda
 /dts/bindings/*/vexriscv*                 @mateusz-holenko @kgugala @pgielda
-/ext/fs/                                  @nashif @wentongwu
```

Comment:
Did you mean to delete this line?

Reply:
was merged in another patch already, removed